### PR TITLE
Add MyMuesli to website showcase

### PR DESCRIPTION
**Pull request by skhameneh**
Add MyMuesli to website showcase

### CODE REVIEW

**Comment by skhameneh** [comment]
This app makes use of features in RC1 in addition to video playback and sensor integration (shake on Android) through custom plugins.  The iOS release is pending.

File: website/src/react-native/showcase.js [hunk 1]
```diff
@@ -71,6 +71,12 @@ var featured = [
     author: 'MovieLaLa'
   },
   {
+    name: 'MyMuesli',
+    icon: 'https://lh3.googleusercontent.com/1dCCeiyjuWRgY-Cnv-l-lOA1sVH3Cn0vkVWWZnaBQbhHOhsngLcnfy68WEerudPUysc=w300-rw',
+    linkPlayStore: 'https://play.google.com/store/apps/details?id=com.mymuesli',
```
Comment (chirag04) [inline website/src/react-native/showcase.js]: use link key here. linkPlayStore is when you have both android and ios.

**Comment by bestander** [comment]
@facebook-github-bot shipit

**Comment by mkonicek** [comment]
Hey @skhameneh were you meaning to add this to the list of featured apps?
http://facebook.github.io/react-native/releases/next/showcase.html

**Pull request closed without merging**